app/services/BackgroundTaskService.js

```diff
@@ -1,11 +1,12 @@
 import BackgroundFetch from 'react-native-background-fetch';
 
+import IntersectService from './IntersectService';
```

Comment:
Can we export the singleton rather than have a singleton as a member of the class?